specification/liftrqumulo/resource-manager/Qumulo.Storage/stable/2022-10-12/Qumulo.Storage.json

```diff
@@ -0,0 +1,728 @@
+            "description": "Marketplace offer is subscribed"
+          },
+          {
+            "name": "Suspended",
+            "value": "Suspended",
+            "description": "Marketplace offer is suspended because of non payment"
+          },
+          {
+            "name": "Unsubscribed",
+            "value": "Unsubscribed",
+            "description": "Marketplace offer is unsubscribed"
+          }
+        ]
+      }
+    },
+    "ProvisioningState": {
+      "type": "string",
+      "description": "Provisioning State of the File system resource",
+      "enum": [
+        "Accepted",
+        "Creating",
+        "Updating",
+        "Deleting",
+        "Succeeded",
+        "Failed",
+        "Canceled",
+        "Deleted",
+        "NotSpecified"
+      ],
+      "x-ms-enum": {
+        "name": "ProvisioningState",
+        "modelAsString": false,
+        "values": [
+          {
+            "name": "Accepted",
+            "value": "Accepted",
+            "description": "File system resource creation request accepted"
+          },
+          {
+            "name": "Creating",
+            "value": "Creating",
+            "description": "File system resource creation started"
+          },
+          {
+            "name": "Updating",
+            "value": "Updating",
+            "description": "File system resource is being updated"
+          },
+          {
+            "name": "Deleting",
+            "value": "Deleting",
+            "description": "File system resource deletion started"
+          },
+          {
+            "name": "Succeeded",
+            "value": "Succeeded",
+            "description": "File system resource creation successful"
+          },
+          {
+            "name": "Failed",
+            "value": "Failed",
+            "description": "File system resource creation failed"
+          },
+          {
+            "name": "Canceled",
+            "value": "Canceled",
+            "description": "File system resource creation canceled"
+          },
+          {
+            "name": "Deleted",
+            "value": "Deleted",
+            "description": "File system resource is deleted"
+          },
+          {
+            "name": "NotSpecified",
+            "value": "NotSpecified",
+            "description": "File system resource state is unknown"
+          }
+        ]
+      }
+    },
+    "UserDetails": {
+      "type": "object",
+      "properties": {
+        "email": {
+          "type": "string",
+          "x-ms-secret": true,
+          "description": "User Email"
+        }
+      },
+      "description": "User Details of Qumulo FileSystem resource",
+      "required": [
+        "email"
+      ]
+    },
+    "StorageSku": {
+      "type": "string",
+      "description": "Storage Sku",
+      "enum": [
+        "Standard",
+        "Performance"
+      ],
+      "x-ms-enum": {
+        "name": "StorageSku",
+        "modelAsString": false,
```

Comment:
>false [](http://example.com/codeflow?start=25&length=5)

Why model as string false? This will cause back-comp issue for client if enum is ever going to change

Reply:
Partner understands the risk that the list of allowed values would not be extensible if modelasstring is set to false. Making a change on this would be a breaking change in future